### DISCUSSION TITLE
Change autocomplete button

### DISCUSSION THREAD

**Discussion by ch4zzy** [community/community#66481] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

How I can change autocomplete copilot button to Enter instead of Tab in VS Code?

**Comment by Akash1134** [comment, marked as answer]
Hi @ch4zzy ,

Have you tried changing the autocomplete keybinding for Copilot in VS Code by modifying the keybindings.json file?

Open Keyboard Shortcuts (JSON) and select the option from the list. This will open the [keybindings.json](https://code.visualstudio.com/docs/getstarted/keybindings) file in the editor. Add the following line to the file and save it:

`{ "key": "enter", "command": "acceptSelectedSuggestion", "when": "suggestWidgetVisible && textInputFocus" }`

#### [Rebinding keyboard shortcuts](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment?tool=vscode#rebinding-keyboard-shortcuts-1)
